Add storage.md section to docs

JamesLDeveloper opened the pull request:
Added new Storage Section to Guide.

Have included steps on how to create and access storage.

tomaswarynyca commented on docs/guide/storage.md:
````diff
@@ -0,0 +1,196 @@
+<cliWindow>
+
+```text {1}
+ $ 	gccli fx public
````
```suggestion
 $ gccli fx public
```

tomaswarynyca commented on docs/guide/storage.md:
````diff
@@ -0,0 +1,196 @@
+<cliWindow>
+
+```text {1}
+ $  gccli fx deploy
````
```suggestion
 $ gccli fx deploy
```

tomaswarynyca commented on docs/guide/storage.md:
````diff
@@ -0,0 +1,196 @@
+```text {1}
+ $  gccli fx deploy
+👷 Verifying configuration...
+📦️ Archs: AMD64,ARM64,ARM
````
```suggestion
📦️ Archs: AMD64
```

tomaswarynyca commented on docs/guide/storage.md:
```diff
@@ -0,0 +1,196 @@
+ $ 	gccli fx public
+👷 Validating inputs...
+🚨 You must keep the public endpoint safe!
+🔗 Any existing endpoint will be revoked
```
```suggestion
```

tomaswarynyca commented on docs/guide/storage.md:
```diff
@@ -0,0 +1,196 @@
+func Handle(req handler.Request) (handler.Response, error) {
+	var err error
+
+	body := req.Body
+	headers := http.Header{
+		"Content-Type": []string{req.Header.Get("Content-Type")},
+	}
+
+	if len(body) == 0 {
+		body = []byte("Hello from GO by GreenCloud!")
+		headers.Set("Content-Type", "text/plain; charset=utf-8")
+	}
+
+	text := []byte("I am a storage Test!")
+
+	// set storage
+	newStorage := base64.StdEncoding.EncodeToString([]byte("{\"admin\":1,\"password\":\"rich\"}"))
+	headers.Set("GC-Storage", newStorage)
+
+	return handler.Response{
+		Body:       []byte(text),
+		StatusCode: http.StatusOK,
+		Header:     headers,
+	}, err
```
```suggestion
	// set storage
	newStorage := base64.StdEncoding.EncodeToString([]byte("{\"admin\":1,\"password\":\"rich\"}"))
	headers.Set("GC-Storage", newStorage)

	return handler.Response{
		Body:       []byte("I am a storage Test!"),
		StatusCode: http.StatusOK,
		Header:     http.Header{
		"Content-Type": []string{"text/plain; charset=utf-8"},
	    },
	}, err
```

tomaswarynyca reviewed (changes requested):
You are showing how to set the storage but never how to get it back.

```
// get storage
storage := req.Header.Get("GC-Storage")
text, _ := base64.StdEncoding.DecodeString(storage)
```

```
// set storage
newStorage := base64.StdEncoding.EncodeToString([]byte({\"user\":\"admin\",\"password\":\"rich\"}"))
headers.Set("GC-Storage", newStorage)
```

JamesLDeveloper commented:
> You are showing how to set the storage but never how to get it back.
> 
> ```
> // get storage
> storage := req.Header.Get("GC-Storage")
> text, _ := base64.StdEncoding.DecodeString(storage)
> ```
> 
> ```
> // set storage
> newStorage := base64.StdEncoding.EncodeToString([]byte({\"user\":\"admin\",\"password\":\"rich\"}"))
> headers.Set("GC-Storage", newStorage)
> ```

I've commited a change, please can you take another look and see if it is ok now?

tomaswarynyca reviewed (changes requested):
Adding examples we've been looking at via Slack

JamesLDeveloper commented:
Added examples in tabs and changed storage size from 16mb to 1mb. Push is showing both a storage.md and Storage.md. Not sure which will actually be updated in the repo. Thanks
